src/microbe_stage/Microbe.Contact.cs

```diff
@@ -76,6 +76,12 @@ public partial class Microbe
     [JsonProperty]
     private int flashPriority;
 
+    /// <summary>
+    ///   This determines how much time is left (in seconds) until this cell can take damage again after becoming
+    ///   invulnerable due to a damage source. This was added to balance pili but might extend to more sources.
+    /// </summary>
+    private float invulnerabilityDuration;
```

Comment:
This should be JsonProperty so that spamming save and load doesn't let you get past the invulnerability time.

Reply:
Added this and tested that indeed you cannot cheese the invulnerability frames now.